### PR TITLE
Various closed group fixes

### PR DESCRIPTION
**Pull request by Bilb**
* fix display name and avatar to be shown when message is coming from a secondary device
* fix show of expiretimer in the group conversation when it is for it.
* fix a bug creating empty conversation when sync closed group message is received on secondary device
* trigger an expiretimer update message to all members when updating a group.
* trigger an expiretimer update message when sending back group details (after a requestGroupInfo)

### CODE REVIEW

File: ts/receiver/dataMessage.ts [hunk 1]
```diff
@@ -430,17 +431,19 @@ export function initIncomingMessage(data: MessageCreationData): MessageModel {
     isRss,
     source,
     serverId,
+    message,
   } = data;
 
   const type = 'incoming';
+  const groupId = message?.group?.id;
```
Comment (Mikunj) [inline ts/receiver/dataMessage.ts:438]: This could actually be an empty byte array, we need to handle that case

File: js/models/conversations.js [hunk 2]
```diff
@@ -1889,10 +1900,25 @@
           window.console.warn('sendGroupInfo invalid pubkey:', recipient);
           return;
         }
-        libsession
+        await libsession
```
Comment (Mikunj) [inline js/models/conversations.js:1903]: should we wrap this in a try-catch instead
```
try {
    sendGroupUpdate
    sendExpirationUpdate
} catch {
    log error
}
```

File: ts/receiver/dataMessage.ts [hunk 3]
```diff
@@ -430,17 +431,27 @@ export function initIncomingMessage(data: MessageCreationData): MessageModel {
     isRss,
     source,
     serverId,
+    message,
   } = data;
 
   const type = 'incoming';
+  if (
```
Comment (Mikunj) [inline ts/receiver/dataMessage.ts:438]: could we just do
```js
const messageGroupId = message?.group?.id;
const groupId = messageGroupId && messageGroupId.length > 0 ? messageGroupId : null;
```

File: ts/session/utils/Protobuf.ts [hunk 4]
```diff
@@ -9,6 +9,12 @@ import ByteBuffer from 'bytebuffer';
 export function convertToTS(object: any): any {
   // No idea why js `ByteBuffer` and ts `ByteBuffer` differ ...
   if (
```
Comment (Mikunj) [inline ts/session/utils/Protobuf.ts:11]: `object instanceof Uint8Array` doesn't work here?

The reason I had done it for `ByteBuffer` below is that `ByteBuffer` in js and `ByteBuffer` in ts are not equivalent.

Comment (Bilb) [reply ts/session/utils/Protobuf.ts:11]: yes it is working with `instanceof`

**Comment by Bilb** [comment]
windows check failed but due to the directory not empty error

**Pull request merged**